.github/workflows/ocwm-reminders-friday-tuesday.yml

```diff
@@ -0,0 +1,64 @@
+name: Send reminders to join the OCWM
+
+on:
+  schedule:
+    - cron: '0 15 * * 5'  # Runs every Friday at 15:00
```

Comment:
This is running every week bet the meeting is monthly. This should run the friday before and the same day of the meeting.

Reply:
The cron jobs will now run as follows 
cron: '0 15 1-7 * 5' #Runs at 15:00, between day 1 and 7 of the month on Friday 
    - cron: '0 15 1-7 * 2' #Runs at 15:00, between day 1 and 7 of the month on Tuesday